app/models/training_module.rb

```diff
@@ -116,6 +116,16 @@ def assessment_results_page
     ModuleItem.where_type(name, 'assessment_results').first
   end
 
+  # @return [ModuleItem]
+  def confidence_intro_page
+    ModuleItem.where_type(name, 'confidence_intro').first
+  end
+
+  # @return [ModuleItem]
+  def confidence_thank_you_page
```

Comment:
Gives me cause to think we need to look at `#last_page` (will we always end a confidence check, I can't remember)